ganache_url: 'http://192.168.1.136:7545'
account_address: '0x039a68e7117F4533DF66B13CBbc8C1f6dbEf0d18'
account_private_key: '0x175fdddcb42f628b8f5fbd137424aa9b0c1119c650143885716de122c536d229'
contract_address: '0x99B84654136e5A8924EaEad34672823C9aBBF7D3'
contract_abi: '[{"anonymous":false,"inputs":[{"indexed":false,"internalType":"address","name":"from","type":"address"},{"indexed":false,"internalType":"string","name":"document","type":"string"},{"indexed":false,"internalType":"uint256","name":"blockNumber","type":"uint256"}],"name":"Result","type":"event"},{"inputs":[{"internalType":"string","name":"documentHash","type":"string"}],"name":"checkProofDocuments","outputs":[],"stateMutability":"nonpayable","type":"function"},{"inputs":[{"internalType":"string[]","name":"documentHashes","type":"string[]"}],"name":"storeProofDocuments","outputs":[],"stateMutability":"nonpayable","type":"function"}]'
contract_bytecode: '608060405233600160006101000a81548173ffffffffffffffffffffffffffffffffffffffff021916908373ffffffffffffffffffffffffffffffffffffffff16021790555034801561005157600080fd5b50610692806100616000396000f3fe608060405234801561001057600080fd5b50600436106100365760003560e01c8063293c29231461003b578063b4115c7714610057575b600080fd5b61005560048036038101906100509190610249565b610073565b005b610071600480360381019061006c91906102ec565b610175565b005b600160009054906101000a900473ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffffffffffffffffffffffffffff163373ffffffffffffffffffffffffffffffffffffffff1614610103576040517f08c379a00000000000000000000000000000000000000000000000000000000081526004016100fa906103bc565b60405180910390fd5b60005b82829050811015610170576000838383818110610126576101256103dc565b5b9050602002810190610138919061041a565b90610143919061049f565b9050436000808381526020019081526020016000208190555050808061016890610537565b915050610106565b505050565b6000828290610184919061049f565b90507f56a563a228126ef95301e2cc755756dc4ed6193cf17e29d015df9de2b9ffe9f7338484600080868152602001908152602001600020546040516101cd949392919061061c565b60405180910390a1505050565b600080fd5b600080fd5b600080fd5b600080fd5b600080fd5b60008083601f840112610209576102086101e4565b5b8235905067ffffffffffffffff811115610226576102256101e9565b5b602083019150836020820283011115610242576102416101ee565b5b9250929050565b600080602083850312156102605761025f6101da565b5b600083013567ffffffffffffffff81111561027e5761027d6101df565b5b61028a858286016101f3565b92509250509250929050565b60008083601f8401126102ac576102ab6101e4565b5b8235905067ffffffffffffffff8111156102c9576102c86101e9565b5b6020830191508360018202830111156102e5576102e46101ee565b5b9250929050565b60008060208385031215610303576103026101da565b5b600083013567ffffffffffffffff811115610321576103206101df565b5b61032d85828601610296565b92509250509250929050565b600082825260208201905092915050565b7f4f6e6c7920746865206f776e6572206f662074686520636f6e7472616374206360008201527f616e2073746f72652074686520646f63756d656e747300000000000000000000602082015250565b60006103a6603683610339565b91506103b18261034a565b604082019050919050565b600060208201905081810360008301526103d581610399565b9050919050565b7f4e487b7100000000000000000000000000000000000000000000000000000000600052603260045260246000fd5b600080fd5b600080fd5b600080fd5b600080833560016020038436030381126104375761043661040b565b5b80840192508235915067ffffffffffffffff82111561045957610458610410565b5b60208301925060018202360383131561047557610474610415565b5b509250929050565b600082905092915050565b6000819050919050565b600082821b905092915050565b60006104ab838361047d565b826104b68135610488565b925060208210156104f6576104f17fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff83602003600802610492565b831692505b505092915050565b7f4e487b7100000000000000000000000000000000000000000000000000000000600052601160045260246000fd5b6000819050919050565b60006105428261052d565b91507fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff8203610574576105736104fe565b5b600182019050919050565b600073ffffffffffffffffffffffffffffffffffffffff82169050919050565b60006105aa8261057f565b9050919050565b6105ba8161059f565b82525050565b82818337600083830152505050565b6000601f19601f8301169050919050565b60006105ec8385610339565b93506105f98385846105c0565b610602836105cf565b840190509392505050565b6106168161052d565b82525050565b600060608201905061063160008301876105b1565b81810360208301526106448185876105e0565b9050610653604083018461060d565b9594505050505056fea26469706673582212206b2e2384d47cb0e5cc56c5d90b8928a5952d65f0b8b4c7330cf8140c8859494a64736f6c63430008120033'
